pallets/ocex/src/tests.rs

```diff
@@ -2363,65 +2369,69 @@ pub fn test_allowlist_with_limit_reaching_returns_error() {
 }
 
 use crate::{
-	settlement::{add_balance, process_trade, sub_balance},
+	settlement::{add_balance, sub_balance},
 	sr25519::AuthorityId,
 	storage::OffchainState,
 };
 use polkadex_primitives::ingress::{HandleBalance, HandleBalanceLimit};
 
-#[test]
-fn test_set_balances_with_bad_origin() {
-	new_test_ext().execute_with(|| {
-		assert_ok!(OCEX::set_exchange_state(RuntimeOrigin::root(), true));
-		let vec_of_balances: Vec<HandleBalance<AccountId32>> = vec![];
-		let bounded_vec_for_alice: BoundedVec<HandleBalance<AccountId>, HandleBalanceLimit> =
-			BoundedVec::try_from(vec_of_balances).unwrap();
-
-		assert_noop!(OCEX::set_balances(RuntimeOrigin::none(), bounded_vec_for_alice), BadOrigin);
```

Comment:
[Method](https://github.com/Polkadex-Substrate/Polkadex/blob/12a2e8a61be8075e0e90567240a5d1fad37082e8/pallets/ocex/src/lib.rs#L125) in trait and [benchmarks](https://github.com/Polkadex-Substrate/Polkadex/blob/12a2e8a61be8075e0e90567240a5d1fad37082e8/pallets/ocex/src/benchmarking.rs#L302-L323) are still there, please cleanup.